code/web/src/modules/user/Profile.js

```diff
@@ -1,19 +1,151 @@
 // Imports
-import React from 'react'
-import PropTypes from 'prop-types'
-import { connect } from 'react-redux'
-import { Helmet } from 'react-helmet'
-import { Link } from 'react-router-dom'
+import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { Helmet } from "react-helmet";
+import { Link } from "react-router-dom";
 
 // UI Imports
-import { Grid, GridCell } from '../../ui/grid'
-import { H3, H4 } from '../../ui/typography'
-import Button from '../../ui/button'
-import { grey, grey2 } from '../../ui/common/colors'
+import { Grid, GridCell } from "../../ui/grid";
+import { H2, H3, H4 } from "../../ui/typography";
+import Button from "../../ui/button";
+import {
+  white,
+  grey,
+  grey2,
+  primaryAccent,
+  secondaryAccent,
+} from "../../ui/common/colors";
+import { Tile } from "../../ui/image";
+import Card from "../../ui/card";
+import { level5 } from "../../ui/common/shadows";
+import { APP_URL } from "../../setup/config/env";
 
 // App Imports
-import userRoutes from '../../setup/routes/user'
-import { logout } from './api/actions'
+import userRoutes from "../../setup/routes/user";
+import { logout } from "./api/actions";
+
+// props.user.details.name
+const mockOrderData = [
+  {
+    user: {
+      details: {
+        id: 1,
+        crateName: "Men's Accesories",
+        deliveryDate: "02/20/2021",
+        orderProducts: [
+          {
+            name: "Belt for Men",
+            returned: false,
+          },
+          {
+            name: "Watch for Men",
+            returned: true,
+          },
+        ],
+        status: "delivered",
+      },
+    },
+  },
+  {
+    user: {
+      details: {
+        id: 2,
+        crateName: "Men's Clothing",
+        deliveryDate: "03/20/2021",
+        orderProducts: [
+          {
+            name: "Shirt for Men",
+            returned: false,
+          },
+          {
+            name: "Pants for Men",
+            returned: false,
+          },
+        ],
+        status: "pending delivery",
+      },
+    },
+  },
+  {
+    user: {
+      details: {
+        id: 3,
+        crateName: "Men's Clothing",
+        deliveryDate: "04/20/2021",
+        orderProducts: [
+          {
+            name: "Shirt for Men",
+            returned: false,
+          },
+        ],
+        status: "pending delivery",
+      },
+    },
+  },
+];
+
+const generateOrderHistory = () => {
+  return mockOrderData.map((order) => {
+    return (
+      <Card
```

Comment:
Nice! What do you think about making this into its own component? Maybe we could use it later for twitter somehow?? If we somehow have time for twitter integration that is. Anyway nice